Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent (props){
+    return(
+        <div className='card__text-content'>
+            <h2>{props.title}</h2>
+            <p>{props.text}</p>
+            <a className="handle" href='https://www.reactjs.org'>{props.handle}</a>
```

Comment:
Good job on passing 3 different props from the parent and using them in the child.